playbooks/tasks/portal-role-task.yml

```diff
@@ -1,5 +1,7 @@
 ---
 
+# xxx This ansible playbook task, should be moved to portal role
+
```

Comment:
Shouldn't this comment be changed now?

Reply:
I just added this comment now.

This file is now an ansible playbook task (not an ansible role task).

Playbook is like a script,
role is a reusable library used by playbooks.

Once it's finished this file (task) should be moved to separate ansible role repo (https://github.com/SkynetLabs/ansible-role-skynet-webportal), where it should belong so Skynet playbooks and community playbooks can execute skynet-webportal role.

It's not yet now part of the role, because during development I would need to update the role task, reinstall the role and run/debug the playbook each time.